proco/connection_statistics/serializers.py

```diff
@@ -39,3 +45,34 @@ class Meta:
             'connectivity_availability',
         )
         read_only_fields = fields
+
+
+class RealTimeConnectivitySerializer(serializers.ModelSerializer):
+    school = SeparatedReadWriteField(write_field=serializers.IntegerField(),
+                                     read_field=serializers.SerializerMethodField())
+
+    class Meta:
+        model = RealTimeConnectivity
+        fields = (
+            'school',
+            'connectivity_speed',
+            'connectivity_latency',
+            'created',
+        )
+        read_only_fields = ('created', )
+
+    def get_school(self, instance):
+        from proco.schools.serializers import BaseSchoolSerializer
+        return BaseSchoolSerializer(instance).data['name']
```

Comment:
why do we need serializer at all? instance.school.name. or even replace read_field with ReadOnlyField(source='school.name')